medicines/web/infrastructure/README.md

```diff
@@ -0,0 +1,36 @@
+# products.mhra.gov.uk Website Infrastucture
+
+Instructions on getting the website set up.
+As steps are completed, the developer completing those steps is responsible for updating this document.
+This document should become a guide for setting up the website infrastructure on a new environment and for connecting to the existing environment.
+
+## Steps
+
+1. Docker time!
+
+   Create a `Dockerfile` in `/medicines/web` which will run `yarn dev`, serving up the Nextjs website.
```

Comment:
Where do I get this from?

Reply:
It has to be written. This documentation is to-be more than an as-is. I can perhaps make this clearer.

Reply:
Agreed.